src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -293,80 +294,24 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
-    auto isSequencePrimitiveSupported = [](const_node_ptr &node) -> bool {
-        const auto& data = node->input(0);
-        const auto& data_pshape = data.get_partial_shape();
-        // WA: dynamic shapes make impossible to check seq_len due to shapeOf subgraphs
-        // but the sequence is still supported in CPU and doesn't need to be decomposed
-        if (data_pshape.is_dynamic())
-            return true;
-        if (data_pshape.rank().is_static() && data_pshape.rank().get_length() > 1 && !data_pshape[1].is_static())
-            return false;
-        auto max_seq_len = data.get_shape().at(1);
-        if (const auto &rnn_seq = std::dynamic_pointer_cast<const ov::opset6::RNNSequence>(node)) {
-            return rnn_seq->get_clip() == 0.0f &&
-                !ov::op::util::is_seq_len_provided(rnn_seq->get_input_node_shared_ptr(2),
-                                                   max_seq_len);
-        } else if (const auto &gru_seq = std::dynamic_pointer_cast<const ov::opset6::GRUSequence>(
-                       node)) {
-            return gru_seq->get_clip() == 0.0f &&
-                gru_seq->get_activations() == std::vector<std::string>{"sigmoid", "tanh"} &&
-                !ov::op::util::is_seq_len_provided(gru_seq->get_input_node_shared_ptr(2),
-                                                   max_seq_len);
-        } else if (const auto &augru_seq = std::dynamic_pointer_cast<const ov::op::internal::AUGRUSequence>(
-                       node)) {
-            return augru_seq->get_clip() == 0.0f &&
-                augru_seq->get_activations() == std::vector<std::string>{"sigmoid", "tanh"} &&
-                !ov::op::util::is_seq_len_provided(augru_seq->get_input_node_shared_ptr(2),
-                                                   max_seq_len);
-        } else if (const auto &lstm_seq = std::dynamic_pointer_cast<const ov::opset6::LSTMSequence>(
-                       node)) {
-            return lstm_seq->get_clip() == 0.0f &&
-                lstm_seq->get_activations() == std::vector<std::string>{"sigmoid", "tanh", "tanh"} &&
-                !ov::op::util::is_seq_len_provided(lstm_seq->get_input_node_shared_ptr(3),
-                                                   max_seq_len);
-        }
-        return false;
-    };
-
     CPU_SET_CALLBACK_COMMON(manager,
-        [isSequencePrimitiveSupported](const_node_ptr &node) -> bool {
-            return isSequencePrimitiveSupported(node);
+        [](const_node_ptr &node) -> bool {
+            const auto& data_pshape = node->get_input_partial_shape(0);
+            // WA: dynamic shapes make impossible to check seq_len due to shapeOf subgraphs
+            // but the sequence is still supported in CPU and doesn't need to be decomposed
+            if (data_pshape.is_dynamic())
+                return true;
+            std::string msg;
+            return node::RNN::isSupportedOperation(node, msg);
```

Comment:
I believe we also need to update functional tests to cover this case with Parameters
maybe these
tests/functional/shared_test_classes/src/single_layer/lstm_cell.cpp

Reply:
Isn't it covered by conformance tests?